cloudfoundry-operations/src/main/java/org/cloudfoundry/operations/services/DefaultServices.java

```diff
@@ -174,8 +188,8 @@ public DefaultServices(CloudFoundryClient cloudFoundryClient, Mono<String> space
         return Mono
             .when(ValidationUtils.validate(request), this.spaceId)
             .flatMap(function((request1, spaceId) -> Optional.ofNullable(request1.getServiceName())
-                .map(serviceName -> getSpaceService(this.cloudFoundryClient, spaceId, serviceName).flux())
-                .orElse(requestListServices(this.cloudFoundryClient, spaceId))
+                    .map(serviceName -> getSpaceService(this.cloudFoundryClient, spaceId, serviceName).flux())
```

Comment:
@LokeshN Incorrect formatting here (big red flag if you're changes are updating existing code using our formatter).  I'll fix.